src/ProposalTypesConfigurator.sol

```diff
@@ -282,13 +282,10 @@ contract ProposalTypesConfigurator is IProposalTypesConfigurator {
             if (calldatas[i].length < 4) revert InvalidCalldata();
 
             bytes24 scopeKey = _pack(targets[i], bytes4(calldatas[i]));
-            if (_assignedScopes[proposalTypeId][scopeKey].length != 0) {
-                validateProposedTx(calldatas[i], proposalTypeId, scopeKey);
-            } else {
-                if (_scopeExists[scopeKey]) {
-                    revert InvalidProposedTxForType();
-                }
+            if (_assignedScopes[proposalTypeId][scopeKey].length == 0) {
+                revert InvalidProposedTxForType();
```

Comment:
As discussed, let's move this logic to a module. Touching scopes here just has the side effect of making the white list the source of truth for any type of execution regardless of proposal type. That means someone could use different quorum rules for the same contract call, and that violates the assumption of scopes unless explicated by the scoping rules.